server/tests/model.specs/favourites.specs.js

```diff
@@ -0,0 +1,69 @@
+import mocha from "mocha";
+import chai from "chai";
+import uuid from "uuid/v4";
+import database from "../../models/index";
+import fixtures from "./model.fixtures";
+
+const sequelize = database.sequelize;
+const assert = chai.assert;
+const userTemplate = fixtures.user;
+const { Favourites, Articles, Users } = database;
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true }).catch(() => {
+    //tables were initialized already
+  });
+});
+
+const favouriteDependencies = async () => {
+  const users = await Users.create(userTemplate);
+  const userId = users.get("id");
+  const articleTemplate = Object.assign(fixtures.article, { authorId: userId });
+  const articles = await Articles.create(articleTemplate);
+  const articleId = articles.get("id");
+
+  return Promise.resolve({
+    userId,
+    articleId,
+    users,
+    articles
+  });
+};
+
+describe("Favourites table model ", () => {
+  it("should create a favourite table with valid userid and articleId", async () => {
+    //Favourites table must be created with two foreign key, userid and articleid
```

Comment:
Please remove all comments your tests should describe themselves. If you have to write comments on your tests then they are not defined properly.